### ISSUE TITLE
Streaming data never reaches client

### ISSUE THREAD

**Issue by watt** [tjanczuk/iisnode#504]
I am running into a problem similar to the one in #386, running a very simple app that streams data continuously until the client disconnects:

```javascript
var server = http.createServer();
server.on('request', function(request, response) {
    var interval = setInterval(function() {
        response.write("some data\r\n");
    }, 1000);
    request.on('close', function() {
        clearInterval(interval);
    });
});
server.listen(config.port);
```

I have set `flushResponse` to true in both the `iisnode.yml` file and in `web.config`.

Logging on the server indicates that node is writing chunks and they are being flushed. But client connections never receive any data. Even the headers do not come through.

I'm running on an Azure App Service.

I've also posted this question on [Server Fault](http://serverfault.com/questions/776673/how-can-i-configure-iis-on-azure-to-stream-chunked-data-from-a-node-js-app) and the [MSDN forums](https://social.msdn.microsoft.com/Forums/azure/en-US/65823c20-9df4-4baf-ba18-090f44424d28/nodejs-how-can-i-configure-iis-to-stream-chunked-data-from-a-nodejs-app?forum=windowsazurewebsitespreview) but have yet to find a fix.

**Comment by markusahlstrand** [comment]
I posted in the issue you references but might as well write here as well. We had a simliar issue and had to set the responseBufferLimit to 0 to make it work:

```
 <handlers>
    <add name="iisnode" path="app.js" verb="*" modules="iisnode" responseBufferLimit="0"/>
 </handlers>
```

**Comment by watt** [comment]
Yeah, I tried setting `responseBufferLimit` as well, but it didn't make a difference.

**Comment by sahilNaran** [comment]
What's the difference between flushResponse="true" and responseBufferLimit="0", both seems to work fine?

**Comment by markusahlstrand** [comment]
I think that the flushResponse only affects the node process whereas the responseBufferLimit sets the outbound buffer to 0 in IIS
